src/Node/Buffer/ST.purs

```diff
@@ -22,26 +49,95 @@ run :: (forall h. ST h (STBuffer h)) -> ImmutableBuffer
 run st = ST.run (st >>= unsafeFreeze)
 
 instance mutableBufferST :: MutableBuffer (STBuffer h) (ST h) where
-  create = Internal.create
-  freeze = Internal.copyAll
-  unsafeFreeze = Internal.unsafeFreeze
-  thaw = Internal.copyAll
-  unsafeThaw = Internal.unsafeThaw
-  fromArray = Internal.fromArray
-  fromString = Internal.fromString
-  fromArrayBuffer = Internal.fromArrayBuffer
-  toArrayBuffer = Internal.toArrayBuffer
-  read = Internal.read
-  readString = Internal.readString
-  toString = Internal.toString
-  write = Internal.write
-  writeString = Internal.writeString
-  toArray = Internal.toArray
-  getAtOffset = Internal.getAtOffset
-  setAtOffset = Internal.setAtOffset
-  slice = Internal.slice
-  size = Internal.size
-  concat = Internal.concat
-  concat' = Internal.concat'
-  copy = Internal.copy
-  fill = Internal.fill
+  create = create
+  freeze = freeze
+  unsafeFreeze = unsafeFreeze
+  thaw = thaw
+  unsafeThaw = unsafeThaw
+  fromArray = fromArray
+  fromString = fromString
+  fromArrayBuffer = fromArrayBuffer
+  toArrayBuffer = toArrayBuffer
+  read = read
+  readString = readString
+  toString = toString
+  write = write
+  writeString = writeString
+  toArray = toArray
+  getAtOffset = getAtOffset
+  setAtOffset = setAtOffset
+  slice = slice
+  size = size
+  concat = concat
+  concat' = concat'
+  copy = copy
+  fill = fill
+
+create ∷ ∀ (h ∷ Region). Int → ST h (STBuffer h)
+create = Internal.create
+
+freeze ∷ ∀ (h ∷ Region). STBuffer h → ST h ImmutableBuffer
+freeze = Internal.copyAll
+
+unsafeFreeze ∷ ∀ (h ∷ Region). STBuffer h → ST h ImmutableBuffer
+unsafeFreeze = Internal.unsafeFreeze
+
+thaw ∷ ∀ (h ∷ Region). ImmutableBuffer → ST h (STBuffer h)
```

Comment:
Why the Unicode?

Reply:
Rather than manually type out each signature, I used the IDE to supply it. But yeah, I forgot to remove this before opening the PR. Latest commit removes the unicode.